site/layouts/courses/section.html

```diff
@@ -1,25 +1,23 @@
 {{ define "main" }}
 {{ $course_home := printf "%s/%s/%s" "courses" .CurrentSection.Params.course_id "_index.md" }}
 {{ if eq .File.Path $course_home }}
-<div class="homepage-content">
-  <div class="large-and-above-only">
-    {{ partial "course_image.html" . }}
-  </div>
-  <div class="container-fluid pt-0 pt-md-4 pb-4">
-    <div class="row medium-and-below-only px-3 pb-2 justify-content-between">
+<div class="homepage-content d-flex">
```

Comment:
The image seems to be pushing all the course home page content to the left.  Should text wrap around the bottom of the image?  I feel like long course descriptions could look kinda awkward with unlimited white space below the image:

![image](https://user-images.githubusercontent.com/12089658/76001512-3182fd80-5ed3-11ea-884d-f6e996a7a52e.png)

Reply:
Maybe one div for the course title and instructor / course number table and the course image, then another below it for the course description?

Reply:
I also see that this is how it looks in the design, so maybe this is a question for @abdulkdawson.  However, since he's on vacation feel free to ignore me and merge it and we can ask him about it later and change it if we decide to do it differently.

Reply:
I was modeling it on the mockups, for instance this one:

![Screenshot from 2020-03-05 11-24-03](https://user-images.githubusercontent.com/6207644/76001926-dac9f380-5ed3-11ea-83f9-44ed165dfdcd.png)

Reply:
Right, yea, I see that now.  It might just be the fact that course images don't have captions yet.  This data exists in master.json, but I'm not importing it into the markdown for some reason.  I'll whip up a quick PR to get it in there so you can implement it here.